examples/demo.py

```diff
@@ -4,7 +4,7 @@
 from PyQt5.QtWidgets import QApplication, QAction
 import numpy as np
 
-from napari_gui.elements import Window
+from napari_gui import Window, Viewer
```

Comment:
btw, Claire McQuinn had a CC0 demo 3D + 4c dataset that she wanted to include in the skimage docs. We could use it for Napari as well. The spherical blobs are not very compelling. Discussion here: https://github.com/scikit-image/scikit-image/issues/2513

Reply:
Thanks for pointing us to that. I think this relates to #84, and I'd rather address in a future PR